src/wasm32/ffi.c

```diff
@@ -344,9 +344,9 @@ ffi_call, (ffi_cif * cif, ffi_fp fn, void *rvalue, void **avalue),
 
 EM_JS_MACROS(void *, ffi_closure_alloc_helper, (size_t size, void **code), {
   var closure = _malloc(size);
-  var func_ptr = getEmptyTableSlot();
-  DEREF_U32(code, 0) = func_ptr;
-  CLOSURE__wrapper(closure) = func_ptr;
+  var index = getEmptyTableSlot();
+  DEREF_U32(code, 0) = index;
+  CLOSURE__wrapper(closure) = index;
```

Comment:
This just changes the name of the variable? I sure hope this didn't fix any bugs. Presumably the actual changes here went away after a rebase...

Reply:
It was related to this error:
```
exception thrown: TypeError: Cannot read property 'delete' of undefined,TypeError: Cannot read property 'delete' of undefined
    at removeFunction (/root/repo/testsuite/test-callback:2:5150)
    at ffi_closure_free_helper (/root/repo/testsuite/test-callback:2:12958)
```

Which could be fixed by swapping `removeFunction` call with `freeTableIndexes.push(index)`. See:
https://github.com/emscripten-core/emscripten/blob/4d864df0a57024d667e8db171aab2c3385d04c30/src/runtime_functions.js#L168-L171

Hence, I renamed `func_ptr` variable to `index`.